### ISSUE TITLE
Pyinstaller Error

### ISSUE THREAD

**Issue by Atisang** [brentvollebregt/auto-py-to-exe#107]
When trying to convert a .py file to .exe using auto-py-to-exe I get the following error :

FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\ATISAN~1.PRE\\AppData\\Local\\Temp\\tmp0v3t65xi\\application\\Significance_Tester\\etc\\jupyter\\kernels\\anaconda3\\Lib\\site-packages\\sklearn\\datasets\\tests\\data\\openml\\40675\\api-v1-json-data-list-data_name-glass2-limit-2-data_version-1-status-deactivated.json.gz'

When using the cmd I get Permission denied error.

**Comment by brentvollebregt** [comment]
What options are you using in the ui?

**Comment by Atisang** [comment]
Hi Brent 

I am using the Onefile option and Window Based(hide the console option)
All other options are checked, which for the Icon, Additional Files, Advanced and Settings.

**Comment by brentvollebregt** [comment]
Could you provide the output displayed in the ui?

**Comment by Atisang** [comment]
Hi Brent,

It just worked.
I ran the pyinstaller in cmd as administrator and it actually ran successfully.

Thanks

**Comment by brentvollebregt** [comment]
The command worked in PyInstaller but not in auto-py-to-exe? Could you provide the auto-py-to-exe output so I can see what went wrong?

**Comment by Atisang** [comment]
Hey Brent,

I just ran it again on auto-to-exe , and it came out successful.
Pretty weird, but yes it worked - the success output is below

Running auto-py-to-exe v2.7.3
Building directory: C:\Users\Gold~1.PRE\AppData\Local\Temp\tmpg0k32xd1
Provided command: pyinstaller --noconfirm --onefile --noconsole  "C:/Users/Gold.PRETORIA/Desktop/Significance Tester/Significance_Tester.py"
Recursion Limit is set to 5000
Executing: pyinstaller --noconfirm --onefile --noconsole C:/Users/Gold.PRETORIA/Desktop/Significance Tester/Significance_Tester.py --distpath C:\Users\Gold~1.PRE\AppData\Local\Temp\tmpg0k32xd1\application --workpath C:\Users\Gold~1.PRE\AppData\Local\Temp\tmpg0k32xd1\build --specpath C:\Users\Gold~1.PRE\AppData\Local\Temp\tmpg0k32xd1

47228 INFO: PyInstaller: 3.6
47233 INFO: Python: 3.7.4 (conda)
47237 INFO: Platform: Windows-10-10.0.19041-SP0
47240 INFO: wrote C:\Users\Gold~1.PRE\AppData\Local\Temp\tmpg0k32xd1\Significance_Tester.spec
47245 INFO: UPX is not available.
47250 INFO: Extending PYTHONPATH with paths
['C:\\Users\\Gold.PRETORIA\\Desktop\\Significance Tester',
 'C:\\Users\\Gold~1.PRE\\AppData\\Local\\Temp\\tmpg0k32xd1']
47253 INFO: checking Analysis
47256 INFO: Building Analysis because Analysis-00.toc is non existent
47259 INFO: Initializing module dependency graph...
47265 INFO: Caching module graph hooks...
47272 INFO: Analyzing base_library.zip ...
51446 INFO: Caching module dependency graph...
51537 INFO: running Analysis Analysis-00.toc
51556 INFO: Adding Microsoft.Windows.Common-Controls to dependent assemblies of final executable
  required by C:\Users\Gold.PRETORIA\AppData\Local\Continuum\anaconda3\python.exe
51824 INFO: Analyzing C:\Users\Atisang.PRETORIA\Desktop\Significance Tester\Significance_Tester.py
52883 INFO: Processing pre-find module path hook   distutils
52886 INFO: distutils: retargeting to non-venv dir 'C:\\Users\\Gold.PRETORIA\\AppData\\Local\\Continuum\\anaconda3\\lib'
54505 INFO: Processing pre-safe import module hook   setuptools.extern.six.moves
55218 INFO: Processing pre-find module path hook   site
55224 INFO: site: retargeting to fake-dir 'C:\\Users\\Gold.PRETORIA\\AppData\\Local\\Continuum\\anaconda3\\lib\\site-packages\\PyInstaller\\fake-modules'
65228 INFO: Processing pre-safe import module hook   six.moves
71666 INFO: Processing pre-safe import module hook   urllib3.packages.six.moves
89725 INFO: Processing module hooks...
89730 INFO: Loading module hook "hook-babel.py"...
89822 INFO: Loading module hook "hook-certifi.py"...
89826 INFO: Loading module hook "hook-cryptography.py"...
90089 INFO: Loading module hook "hook-distutils.py"...
90097 INFO: Loading module hook "hook-docutils.py"...
92065 INFO: Loading module hook "hook-encodings.py"...
92128 INFO: Loading module hook "hook-importlib_metadata.py"...
92133 INFO: Loading module hook "hook-IPython.py"...
92639 INFO: Excluding import 'PyQt4'
92648 INFO:   Removing import of PyQt4 from module IPython.external.qt_loaders
92653 INFO: Excluding import 'gtk'
92662 INFO: Excluding import 'tkinter'
92671 INFO:   Removing import of tkinter from module IPython.lib.clipboard
92675 INFO: Excluding import 'matplotlib'
92685 INFO:   Removing import of matplotlib.figure from module IPython.core.pylabtools
92688 INFO:   Removing import of matplotlib.pyplot from module IPython.core.pylabtools
92691 INFO:   Removing import of matplotlib from module IPython.core.pylabtools
92696 INFO:   Removing import of matplotlib._pylab_helpers from module IPython.core.pylabtools
92703 INFO: Excluding import 'PyQt5'
92713 INFO:   Removing import of PyQt5.QtSvg from module IPython.external.qt_loaders
92718 INFO:   Removing import of PyQt5.QtWidgets from module IPython.external.qt_loaders
92722 INFO:   Removing import of PyQt5 from module IPython.external.qt_loaders
92727 INFO:   Removing import of PyQt5.QtGui from module IPython.external.qt_loaders
92732 INFO:   Removing import of PyQt5.QtCore from module IPython.external.qt_loaders
92737 INFO: Excluding import 'PySide'
92747 INFO:   Removing import of PySide from module IPython.external.qt_loaders
92752 INFO: Loading module hook "hook-jedi.py"...
92906 INFO: Loading module hook "hook-jinja2.py"...
92931 INFO: Loading module hook "hook-jsonschema.py"...
92941 INFO: Loading module hook "hook-lib2to3.py"...
92949 INFO: Loading module hook "hook-matplotlib.backends.py"...
93566 INFO:   Matplotlib backend "GTK3Agg": ignored
    cairo backend requires that pycairo>=1.11.0 or cairocffiis installed
93882 INFO:   Matplotlib backend "GTK3Cairo": ignored
    cairo backend requires that pycairo>=1.11.0 or cairocffiis installed
94308 INFO:   Matplotlib backend "MacOSX": ignored
    cannot import name '_macosx' from 'matplotlib.backends' (C:\Users\Gold.PRETORIA\AppData\Local\Continuum\anaconda3\lib\site-packages\matplotlib\backends\__init__.py)
95023 INFO:   Matplotlib backend "nbAgg": added
95613 INFO:   Matplotlib backend "Qt4Agg": added
95899 INFO:   Matplotlib backend "Qt4Cairo": ignored
    cairo backend requires that pycairo>=1.11.0 or cairocffiis installed
96329 INFO:   Matplotlib backend "Qt5Agg": added
96607 INFO:   Matplotlib backend "Qt5Cairo": ignored
    cairo backend requires that pycairo>=1.11.0 or cairocffiis installed
97011 INFO:   Matplotlib backend "TkAgg": added
97404 INFO:   Matplotlib backend "TkCairo": ignored
    cairo backend requires that pycairo>=1.11.0 or cairocffiis installed
97825 INFO:   Matplotlib backend "WebAgg": added
98199 INFO:   Matplotlib backend "WX": ignored
    No module named 'wx'
98487 INFO:   Matplotlib backend "WXAgg": ignored
    No module named 'wx'
98777 INFO:   Matplotlib backend "WXCairo": ignored
    No module named 'wx'
99074 INFO:   Matplotlib backend "agg": added
99355 INFO:   Matplotlib backend "cairo": ignored
    cairo backend requires that pycairo>=1.11.0 or cairocffiis installed
99731 INFO:   Matplotlib backend "pdf": added
100112 INFO:   Matplotlib backend "pgf": added
100406 INFO:   Matplotlib backend "ps": added
100691 INFO:   Matplotlib backend "svg": added
101046 INFO:   Matplotlib backend "template": added
101439 INFO: Loading module hook "hook-matplotlib.py"...
101686 INFO: Loading module hook "hook-nbconvert.py"...
101706 INFO: Loading module hook "hook-nbformat.py"...
101724 INFO: Loading module hook "hook-notebook.py"...
112711 INFO: Loading module hook "hook-numpy.core.py"...
112821 INFO: Loading module hook "hook-numpy.py"...
112826 INFO: Loading module hook "hook-PIL.Image.py"...
113276 INFO: Loading module hook "hook-PIL.py"...
113284 INFO: Excluding import 'PyQt4'
113292 INFO:   Removing import of PyQt4 from module PIL.ImageQt
113296 INFO: Excluding import 'tkinter'
113307 INFO:   Removing import of tkinter from module PIL.ImageTk
113312 INFO: Import to be excluded not found: 'FixTk'
113317 INFO: Excluding import 'PyQt5'
113327 INFO:   Removing import of PyQt5.QtGui from module PIL.ImageQt
113332 INFO:   Removing import of PyQt5.QtCore from module PIL.ImageQt
113338 INFO: Excluding import 'PySide'
113345 INFO:   Removing import of PySide from module PIL.ImageQt
113351 INFO: Loading module hook "hook-PIL.SpiderImagePlugin.py"...
113358 INFO: Excluding import 'tkinter'
113363 INFO: Import to be excluded not found: 'FixTk'
113372 INFO: Loading module hook "hook-pkg_resources.py"...
113929 INFO: Processing pre-safe import module hook   win32com
114358 INFO: Excluding import '__main__'
114366 INFO:   Removing import of __main__ from module pkg_resources
114372 INFO: Loading module hook "hook-pycparser.py"...
114377 INFO: Loading module hook "hook-pydoc.py"...
114391 INFO: Loading module hook "hook-pygments.py"...
115935 INFO: Loading module hook "hook-PyQt5.py"...
116306 INFO: Loading module hook "hook-PyQt5.QtCore.py"...
116343 INFO: Loading module hook "hook-PyQt5.QtGui.py"...
116363 INFO: Loading module hook "hook-PyQt5.QtSvg.py"...
116384 INFO: Loading module hook "hook-PyQt5.QtWidgets.py"...
116409 INFO: Loading module hook "hook-pytest.py"...
117355 INFO: Loading module hook "hook-pythoncom.py"...
117827 INFO: Loading module hook "hook-pytz.py"...
117905 INFO: Loading module hook "hook-pywintypes.py"...
118366 INFO: Loading module hook "hook-scipy.linalg.py"...
118372 INFO: Loading module hook "hook-scipy.py"...
118387 INFO: Loading module hook "hook-scipy.sparse.csgraph.py"...
118396 INFO: Loading module hook "hook-scipy.special._ellip_harm_2.py"...
118402 INFO: Loading module hook "hook-scipy.special._ufuncs.py"...
118411 INFO: Loading module hook "hook-setuptools.py"...
119191 INFO: Loading module hook "hook-sphinx.py"...
123686 INFO: Loading module hook "hook-sqlite3.py"...
123780 INFO: Loading module hook "hook-sysconfig.py"...
123786 INFO: Loading module hook "hook-win32com.py"...
124449 INFO: Loading module hook "hook-xml.dom.domreg.py"...
124455 INFO: Loading module hook "hook-xml.etree.cElementTree.py"...
124464 INFO: Loading module hook "hook-xml.py"...
124470 INFO: Loading module hook "hook-zmq.py"...
127671 WARNING: Hidden import "such" not found!
127678 WARNING: Hidden import "or" not found!
127685 WARNING: Hidden import "_cffi_ext.c" not found!
127690 WARNING: Hidden import "error" not found!
127805 WARNING: Hidden import "file:" not found!
127812 WARNING: Hidden import "Cannot" not found!
127818 WARNING: Hidden import "open" not found!
127825 WARNING: Hidden import "'sys/un.h':" not found!
127832 WARNING: Hidden import "fatal" not found!
127838 WARNING: Hidden import "directory" not found!
127846 WARNING: Hidden import "file" not found!
127939 WARNING: Hidden import "No" not found!
127946 WARNING: Hidden import "C:\Users\Gold.PRETORIA\AppData\Local\Continuum\anaconda3\lib\site-packages\zmq\backend\cffi\__pycache__\_cffi_ext.c(213):" not found!
127955 WARNING: Hidden import "C1083:" not found!
127959 INFO: Loading module hook "hook-_tkinter.py"...
128119 INFO: checking Tree
128125 INFO: Building Tree because Tree-00.toc is non existent
128133 INFO: Building Tree Tree-00.toc
128251 INFO: checking Tree
128258 INFO: Building Tree because Tree-01.toc is non existent
128267 INFO: Building Tree Tree-01.toc
128293 INFO: Loading module hook "hook-gevent.py"...
128490 INFO: Determining a mapping of distributions to packages...
182861 WARNING: Unable to find package for requirement greenlet from package gevent.
182866 INFO: Packages required by gevent:
['cffi']
184507 INFO: Loading module hook "hook-lxml.etree.py"...
184930 INFO: Looking for ctypes DLLs
185049 INFO: Analyzing run-time hooks ...
185072 INFO: Including run-time hook 'pyi_rth_pkgres.py'
185080 INFO: Including run-time hook 'pyi_rth_win32comgenpy.py'
185090 INFO: Including run-time hook 'pyi_rth_multiprocessing.py'
185099 INFO: Including run-time hook 'pyi_rth_traitlets.py'
185107 INFO: Including run-time hook 'pyi_rth_certifi.py'
185115 INFO: Including run-time hook 'pyi_rth_pyqt5.py'
185121 INFO: Including run-time hook 'pyi_rth_mplconfig.py'
185129 INFO: Including run-time hook 'pyi_rth_mpldata.py'
185137 INFO: Including run-time hook 'pyi_rth__tkinter.py'
185182 INFO: Looking for dynamic libraries
185248 WARNING: lib not found: mpich2mpi.dll dependency of C:\Users\Atisang.PRETORIA\AppData\Local\Continuum\anaconda3\Library\bin\mkl_blacs_mpich2_ilp64.dll
185282 WARNING: lib not found: mpich2mpi.dll dependency of C:\Users\Atisang.PRETORIA\AppData\Local\Continuum\anaconda3\Library\bin\mkl_blacs_mpich2_lp64.dll
185315 WARNING: lib not found: impi.dll dependency of C:\Users\Atisang.PRETORIA\AppData\Local\Continuum\anaconda3\Library\bin\mkl_blacs_intelmpi_ilp64.dll
185520 WARNING: lib not found: impi.dll dependency of C:\Users\Atisang.PRETORIA\AppData\Local\Continuum\anaconda3\Library\bin\mkl_blacs_intelmpi_lp64.dll
185966 WARNING: lib not found: pgf90.dll dependency of C:\Users\Atisang.PRETORIA\AppData\Local\Continuum\anaconda3\Library\bin\mkl_pgi_thread.dll
185976 WARNING: lib not found: pgc14.dll dependency of C:\Users\Atisang.PRETORIA\AppData\Local\Continuum\anaconda3\Library\bin\mkl_pgi_thread.dll
185984 WARNING: lib not found: pgf90rtl.dll dependency of C:\Users\Atisang.PRETORIA\AppData\Local\Continuum\anaconda3\Library\bin\mkl_pgi_thread.dll
188481 INFO: Looking for eggs
188487 INFO: Using Python library C:\Users\Atisang.PRETORIA\AppData\Local\Continuum\anaconda3\python37.dll
188498 INFO: Found binding redirects: 
[]
188532 INFO: Warnings written to C:\Users\ATISAN~1.PRE\AppData\Local\Temp\tmpg0k32xd1\build\Significance_Tester\warn-Significance_Tester.txt
188880 INFO: Graph cross-reference written to C:\Users\ATISAN~1.PRE\AppData\Local\Temp\tmpg0k32xd1\build\Significance_Tester\xref-Significance_Tester.html
192145 INFO: checking PYZ
192153 INFO: Building PYZ because PYZ-00.toc is non existent
192161 INFO: Building PYZ (ZlibArchive) C:\Users\ATISAN~1.PRE\AppData\Local\Temp\tmpg0k32xd1\build\Significance_Tester\PYZ-00.pyz
195765 INFO: Building PYZ (ZlibArchive) C:\Users\ATISAN~1.PRE\AppData\Local\Temp\tmpg0k32xd1\build\Significance_Tester\PYZ-00.pyz completed successfully.
196049 INFO: checking PKG
196055 INFO: Building PKG because PKG-00.toc is non existent
196066 INFO: Building PKG (CArchive) PKG-00.pkg
590875 INFO: Building PKG (CArchive) PKG-00.pkg completed successfully.
594145 INFO: Bootloader C:\Users\Atisang.PRETORIA\AppData\Local\Continuum\anaconda3\lib\site-packages\PyInstaller\bootloader\Windows-64bit\runw.exe
594152 INFO: checking EXE
594163 INFO: Building EXE because EXE-00.toc is non existent
594173 INFO: Building EXE from EXE-00.toc
594180 INFO: Appending archive to EXE C:\Users\ATISAN~1.PRE\AppData\Local\Temp\tmpg0k32xd1\application\Significance_Tester.exe
595886 INFO: Building EXE from EXE-00.toc completed successfully.

Moving project to: C:\Users\Golsd.PRETORIA\OneDrive - Ask Afrika (Pty) Ltd\Documents\Significance Testing Tool\auto-py-to-exe-master\auto-py-to-exe-master\output
Complete.
